arch/arc/core/isr_wrapper.S

```diff
@@ -279,8 +262,21 @@ SECTION_FUNC(TEXT, _isr_demux)
 	bl sys_trace_isr_enter
 #endif
 	/* cannot be done before this point because we must be able to run C */
-	/* r0 is available to be stomped here, and exit_tickless_idle uses it */
-	exit_tickless_idle
+	/* exit tickless idle */
```

Comment:
Sorry for that last minute request, but can we move `/* exit tickless idle */` comment under `#ifdef CONFIG_SYS_POWER_MANAGEMENT` and also switch from `#if defined` to more usual `#ifdef`?

Reply:
I found there're several places that use `#ifdef` in isr_wrapper.s, so I put the changes in a seperate commit.